sdk/core/azure-core/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 - [[#5622]](https://github.com/Azure/azure-sdk-for-cpp/pull/5622) Documentation fix for building the SDK with specific OpenSSL version.  (A community contribution, courtesy of _[ByteYue](https://github.com/ByteYue)_)
 - [[#5515]](https://github.com/Azure/azure-sdk-for-cpp/issues/5515) Add a `ShouldRetry` virtual method to the retry policy to enable customization of service-specific retry logic.
+- Updated JSON library to 3.11.3.
```

Comment:
We shipped this beta out already.  Move to the `Other changes` section of `1.13.0-beta.2 (Unreleased)`

Reply:
good point, done